internal/webhooks/cluster_test.go

```diff
@@ -46,7 +46,8 @@ func TestClusterDefaultNamespaces(t *testing.T) {
 		},
 	}
 	webhook := &Cluster{}
-	t.Run("for Cluster", customDefaultValidateTest(ctx, c, webhook))
+	testFunc := customDefaultValidateTest(ctx, c, webhook)
```

Comment:
Without this change, the `thelper` linter panics. While we can alternatively drop the linter, I would prefer this workaround and keeping it

Reply:
I'll open an issue in the thelper linter